### PR TITLE
Improve rollback cleanup

### PR DESCRIPTION
**Pull request by pirat89**
- removes files of target kernel when --clean option is used after the rollback
- remove /boot/rollback and /boot/manualcleanup dirs when --clean is used
- add /boot/rollback/do_rollback script that will do rollback from RHEL 7

### CODE REVIEW

**Comment by pirat89** [comment]
~~I apologize that I havn't time to test it. So in case you find an issue, feel free to fix it or wait to my pre-testing. But I will be really glad for help here as I have rush days and will be able to test it properly tomorrow. Thanks~~

File: redhat-upgrade-tool.py [hunk 1]
```diff
@@ -316,6 +319,21 @@ def main(args):
...
         print _("getting boot images...")
         kernel, initrd = f.download_boot_images() # TODO: force arch?
+        if args.snapshot_root_lv:
+            # In case of rollback, we want to be able to remove kernel files
+            # of target (upgraded) system after the rollback and be sure that
+            # we will not remove those that possibly exists
+            kv = kernelver(kernel)
+            if not kv:
+                print _("Warning: cannot determine version of target kernel."
+                        " In case of rollback, you will need to removed kernel"
```
Comment (mfranczy) [inline redhat-upgrade-tool.py]: typo, fix to ```you will need to remove kernel``` or ```you will have to```

File: redhat_upgrade_tool/rollback/__init__.py [hunk 2]
```diff
@@ -1,2 +1,10 @@
+
+import os
+
 rhel6_profile = "98c3edb"
 snapshot_metadata_file = "/boot/grub/snapshot.metadata"
+rollback_dir = "/boot/rollback"
+active_kernel_file = os.path.join(rollback_dir, '.active-kernel')
+all_kernels_file = os.path.join(rollback_dir, '.all-kernels')
+target_kernel_file = os.path.join(rollback_dir, '.target-kernel')
+snap_boot_files_file = os.path.join(rollback_dir, '.snap_boot_files')
```
Comment (mfranczy) [inline redhat_upgrade_tool/rollback/__init__.py:10]: I would change the name of this variable, ```snap_boot_files_file```, I see why it's like this, although if someone is looking after a long time into that code, they might be confused.

But it is only a suggestion, up to you..

**Comment by mfranczy** [comment]
I added two minor things, it should work (didn't test it yet).

File: redhat_upgrade_tool/rollback/__init__.py (hunk 2 above)
Comment (pirat89) [reply redhat_upgrade_tool/rollback/__init__.py:10]: I agree and I spent a time above this name already. But I still can't come up with any good name at all, soI will keep this for now.

**Comment by pirat89** [comment]
Related #45

**Review by mfranczy — DISMISSED** [review]

**Review by mfranczy — APPROVED** [review]

**Pull request merged**